src/services/Store/BalanceService.tsx

```diff
@@ -32,6 +33,14 @@ export const toBigNumberJS = (balances: EthScanBalanceMap): BalanceMap => {
   );
 };
 
+export const nestedToBigNumberJS = (
+  balances: EthScanBalanceMap<EthScanBalanceMap>
+): BalanceMap<BalanceMap> => {
+  return Object.fromEntries(
+    Object.keys(balances).map((key) => [key, toBigNumberJS(balances[key])])
```

Comment:
1. use `bigify` to harmonise out big number handling. 
2. Why not return an object instead of a tuple?
3. If so this can be simplified to
```
 .then(mapObjIndexed(bigify))
```

Reply:
We can't use `bigify` due to a mismatch in types of BN. See your comment on line 180.

Reply:
Did simplify the helper functions though!